src/main/java/uk/gov/hmcts/reform/pip/data/management/config/PublicationConfiguration.java

```diff
@@ -5,8 +5,9 @@
  */
 public final class PublicationConfiguration {
 
-    public static final String PROVENANCE_HEADER = "x-provenance";
-    public static final String SOURCE_ARTEFACT_ID_HEADER = "x-source-artefact-id";
+    public static final String PROVENANCE_HEADER = "x-provenance is mandatory however an empty value is provided";
+    public static final String SOURCE_ARTEFACT_ID_HEADER = "x-source-artefact-id is mandatory however an empty value "
+        + "is provided";
```

Comment:
these arent representing strings to display to the user, these are the header attributes that are needed from the API request so these would need changing back